### PR TITLE
PieChart: Make pie gradient more subtle to match other charts

### LINKED ISSUE #36141: PieChart: Make pie slice gradient more subtle to match other visualizations feel
**What happened**:
I updated from 7 series to 8.0.3, and changed an existing pie chart to the new piechart graph, and noticed that the colors, even though I selected the same across graphs, are different in the pie chart.

example

Both reds are the default red color swatch, but the pie chart is #d20f29 and the bar graph is #e8475a.

**What you expected to happen**:
Colors should be the same across graphs

**How to reproduce it (as minimally and precisely as possible)**:
Create a time series graph and a pie chart graph, set an element to use the default red color, the colors are not the same.

**Environment**:
- Grafana version: 8.0.3

### PR DESCRIPTION
**Pull request by nikki-kiga**
Closes #36141 

#### Before PieChart and BarChart
beforeDarkChart

#### After PieChart and BarChart
afterDarkChart

#### Before PieCharts in Light Mode
beforeLightPies

#### After PieCharts in Light Mode
afterLightPies

### CODE REVIEW

File: public/app/plugins/panel/piechart/PieChart.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ import { css } from '@emotion/css';
 import { useComponentInstanceId } from '@grafana/ui/src/utils/useComponetInstanceId';
 import { getTooltipContainerStyles } from '@grafana/ui/src/themes/mixins';
 import { selectors } from '@grafana/e2e-selectors';
+import { darken, lighten } from '@grafana/data/src/themes/colorManipulator';
```
Comment (ryantxu) [inline public/app/plugins/panel/piechart/PieChart.tsx:33]:
```suggestion
import { colorManipulator } from '@grafana/data';
```

and then use it as `colorManipulator.lighten|darken`

When working in the `public/app/plugins/...` it does not really matter, but we should use the standard export paths when possible

**Review by ryantxu — APPROVED** [review]

**Review by torkelo — CHANGES_REQUESTED** [review]
The gradient as is is based on gradient used in stat panels , so want to preserve it to maintain consistency with stat panel. Maybe it can also be tweaked? But the hue spin is an important aspect to the gradient

**Comment by torkelo** [comment]
Maybe just reduce the hue spin a bit ?

**Comment by nikki-kiga** [comment]
Reduced the spin  so that it still matches the stat panel:
reducedSpin

**Review by torkelo — APPROVED** [review]

**Pull request merged**